### PR TITLE
add service component

### PR DESCRIPTION
**Pull request by thanhbui27**

### CODE REVIEW

**Comment by thanhbui27** [comment]
image

File: src/App.tsx [hunk 1]
```diff
@@ -9,6 +10,7 @@ const App = () => {
         <Header />
```
Comment (baonguyen26) [inline src/App.tsx:10]: pls create a page

File: src/components/Service/components/CardService/CardServiceItem.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import TagItem from "../TagService/TagItem";
+import { TDataCard } from "../constants";
+
+type TPropsCard = {
+  item: TDataCard;
+};
+
+const CardServiceItem: React.FC<TPropsCard> = ({ item }) => {
```
Comment (baonguyen26) [inline src/components/Service/components/CardService/CardServiceItem.tsx:8]: destructor

File: src/components/Service/components/TagService/TagItem.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { TDataTag } from "../constants";
+
+type TPropsTagITem = {
+  item: TDataTag;
+  classNames?: string;
```
Comment (baonguyen26) [inline src/components/Service/components/TagService/TagItem.tsx:5]: should have default props for this one

Not: `button-undefined`

**Pull request merged**